project/api/models.py

```diff
@@ -263,11 +263,11 @@ def check_visual_center(sender, instance, **kwargs):  # pylint: disable=W0613
     """
     atomic_set = instance.territory.all()
 
-    if atomic_set.exists():
+    #if atomic_set.exists():
 
-        combined = atomic_set.aggregate(models.Collect("geom"))
-        if combined["geom__collect"].contains(instance.visual_center) is False:
-            raise ValidationError("Visual center is not inside the territory")
+        #combined = atomic_set.aggregate(models.Collect("geom"))
+        #if combined["geom__collect"].contains(instance.visual_center) is False:
+        #    raise ValidationError("Visual center is not inside the territory")
```

Comment:
In the past we've removed blocks of code as opposed to commenting them out to avoid confusion for newcomers. This code will be present in the commit history should we decide to re-implement it later.

Reply:
I would agree with you, it is better to remove unused blocks of code.

But I am not sure, should we perform some checks on visual center or not.

Reply:
On Slack, we have come to the consensus that this will be removed as it interferes with `manage.py loaddata` and the representative point of a nation is not always included within the bounds of its geometry (such as in Great Britain)